helixbass/grouped-ordering
# Instantiate macro

In this PR:
- per https://github.com/helixbass/grouped-ordering/pull/1#discussion_r1440695900, add a probably unnecessary ergonomic macro for creating new grouped-ordering-specification instances

To test:
Per the added test you should be able to use a macro (whose name is the snake-case version of the specified grouped-ordering type name after usage of the `grouped_ordering!()` macro) if you want to manually specify grouped-ordering instances 

Based on `grouped-ordering-trait`
